test/org/sagebionetworks/bridge/services/ScheduledActivityOperationsTest.java

```diff
@@ -0,0 +1,142 @@
+package org.sagebionetworks.bridge.services;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import org.sagebionetworks.bridge.models.schedules.ScheduledActivity;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+
+public class ScheduledActivityOperationsTest {
```

Comment:
Added ScheduledActivityServiceMockTest.orderActivitieFiltersAndSorts() to test filtering and sorting.